### ISSUE TITLE
Slight inconsistency in length? bitset

### ISSUE THREAD

**Issue by meijeru** [red/red#2609]
```
>> length? charset []
== 8
>> length? charset [not]
== 0
>> length? make bitset! #{}
== 0
```
It turns out that a bitset made by `charset` without any bit positions set, does still occupy 8 bits. At first I thought it was not possible to make a bitset of length 0, but there are two ways of doing this, as shown.

**Comment by 9214** [comment]
`bitset!` can also be constructed from `binary!` and `string!`:

```red
>> length? charset []
== 8
>> length? charset ""
== 8
>> length? charset #{}
== 0
```

A few bitset tests expect `make bitset! #{00}` of length `8` as a result of the first two examples. One can wonder if forbidding bitset of length `0` is more consistent.

**Comment by Haxerking** [comment]
Yes for sure but when it will be {100} then it overflow as I see in the CMT
box...


Haxer

**Comment by meijeru** [comment]
What "will be {100}"? There is no binary literal #{100}, one should write #{0100}. And what is the CMT box?

**Comment by 9214** [comment]
@meijeru ☝️ do you have any thoughts on my recent comment?

**Comment by meijeru** [comment]
I am for length 0 in all three cases.

**Comment by GiuseppeChillemi** [comment]
I am for 0 in alle three cases as it is visually coherent with concept of empty content of parens and strings.
